embedly/views.py

```diff
@@ -0,0 +1,23 @@
+"""Embed.ly views"""
+from urllib.parse import unquote
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from django.conf import settings
+
+from embedly.api import get_embedly
+
+
+class EmbedlyViews(APIView):
+    """
+    View for getting embedly API
+    """
+
+    def get(self, request, *args, **kwargs):  # pylint: disable=unused-argument
+        """get Embedly API, return the JSON"""
+        if settings.EMBEDLY_KEY:
+            response = get_embedly(unquote(unquote(self.kwargs["url"])))
```

Comment:
Why does this have to be unquoted twice?

Reply:
it actually doesn't have to be, things seem to work fine if it is only unquoted once. But in order to pass a full URL as a URL parameter I had to call `encodeURIComponent` twice on the frontend, calling `unquote` twice just makes the URL we pass to the `get_embedly` function look like a normal URL again, if that makes sense. I may be missing some easier way to deal with passing a URL as a URL param, haha.